tests/integration/test_core.py

```diff
@@ -133,3 +109,78 @@ async def test_wordpress_functionality(
             admin_username="admin",
             admin_password=default_admin_password
         )
+
+
+@pytest.mark.asyncio
+async def test_wordpress_default_themes(
+        unit_ip_list,
+        get_theme_list_from_ip
+):
+    """
+    arrange: after WordPress charm has been deployed and db relation established
+    act: check installed WordPress themes
+    assert: All default themes should be installed
+    """
+    for unit_ip in unit_ip_list:
+        assert (
+                set(WordpressCharm._WORDPRESS_DEFAULT_THEMES) ==
+                set(get_theme_list_from_ip(unit_ip))
+        ), "default themes installed should match default themes defined in WordpressCharm"
+
+
+@pytest.mark.asyncio
+async def test_wordpress_install_uninstall_themes(
```

Comment:
Would it be worth also having a test where a config change results in both installing a theme and uninstalling a theme? What about installing multiple themes? Could potentially be a parametrised version of this test

Reply:
Fixed, thanks!